components/FButton.vue

```diff
@@ -5,10 +5,10 @@ component.FButton(
   :style="style"
   :class="classes"
   v-bind="specificProps"
-  @blur.stop="emit('blur')"
-  @click.stop="onClick($event, true)"
-  @focus.stop="emit('focus')"
-  @keydown.enter.space.stop="onClick($event, false)"
+  @blur="emit('blur', $event)"
+  @click="handleClick($event, true)"
+  @focus="emit('focus', $event)"
+  @keydown.enter.space="handleClick($event, false)"
```

Comment:
```suggestion
  @keydown.enter.space="handleTrigger($event, false)"
```
Maybe more suitable for both mouse and keyboard event?

Reply:
I'm not sure. In fact, the keyboard simulates a click event - for people that cannot use a mouse. I think "trigger" is not very common and maybe more misleading